apps/test-bundles/webpackUtils.js

```diff
@@ -0,0 +1,100 @@
+const path = require('path');
+const fs = require('fs');
+const resources = require('@uifabric/build/webpack/webpack-resources');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+function createWebpackConfig(entries, includeStats = true) {
+  return Object.keys(entries).map(entryName => {
+    let anaylizerPluginOptions = {
+      analyzerMode: 'static',
+      reportFilename: entryName + '.stats.html',
+      openAnalyzer: false,
+      generateStatsFile: false,
+      logLevel: 'warn',
+    };
+
+    if (includeStats) {
+      anaylizerPluginOptions = {
+        ...anaylizerPluginOptions,
+        generateStatsFile: true,
+        statsOptions: {
+          // https://webpack.js.org/configuration/stats
+          assets: true,
+          modules: true,
+
+          builtAt: false,
+          outputPath: false,
+          namedChunkGroups: false,
+          logging: false,
+          children: false,
+          source: false,
+          reasons: false,
+          chunks: false,
+          cached: false,
+          cachedAssets: false,
+          performance: false,
+          timings: false,
+        },
+        statsFilename: entryName + '.stats.json',
+      };
+    }
+
+    const config = resources.createConfig(
+      entryName,
+      true,
+      {
+        entry: {
+          [entryName]: entries[entryName],
+        },
+        externals: {
+          react: 'React',
+          'react-dom': 'ReactDOM',
+        },
+        plugins: [new BundleAnalyzerPlugin(anaylizerPluginOptions)],
```

Comment:
This is a slight tweak that is definitely not necessary: 

Since this is the only part of each entry that could change via `includeStats` you could pass anaylizerPluginOptions in via each entry. That means you only need 1 webpack file and you would configure the react-next entry to pass the "no-stats" options through.

Reply:
i thought about this. but i went with having 2 configs (as i thought that's cleaner). let me double check

Reply:
fixed!